Makefile

```diff
@@ -22,15 +22,15 @@ test-only:
 
 build-operator: test
 	@echo "-> Building kubi operator"
-	CGO_ENABLED=0 GOARCH=amd64 go build -a -ldflags '-extldflags "-static"' -v -o ./build/kubi-operator -i $(GOPATH)/src/$(REPO)/cmd/operator/main.go
+	CGO_ENABLED=0 GOARCH=amd64 go build -a -ldflags '-extldflags "-static"' -v -o ./build/kubi-operator $(GOPATH)/src/$(REPO)/cmd/operator/main.go
```

Comment:
While we are at it, we should remove the goarch, ldflags, the GOPATH/src/REPO pattern... Pondering why this hasn't been merged though...